### PR TITLE
SQL: date_format aripthmetic expression

### PR DESCRIPTION
**Pull request by unrealwork**
# SQL: date_format arithmetic expression
- [x] tests

### CODE REVIEW

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java [hunk 1]
```diff
@@ -83,6 +63,38 @@ public static void assertTableRowsExist(List<List<String>> expectedRows, StringT
...
+        fail(format(message, expected, actual));
+    }
+
+    private static String format(String message, Object expected, Object actual) {
+        String formatted = "";
+        if (message != null) {
+            formatted = message + " ";
+        }
+
+        return formatted + "expected:<" + expected + "> but was:<" + actual + ">";
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: what about String.format()?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: Are you missing?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: That's a fair point

**Pull request merged**